examples/00-fluent/external_compressible_flow.py

```diff
@@ -255,89 +251,82 @@
 # Define materials
 # ~~~~~~~~~~~~~~~~
 
-solver.tui.define.materials.change_create(
-    "air",
-    "air",
-    "yes",
-    "ideal-gas",
-    "no",
-    "no",
-    "yes",
-    "sutherland",
-    "three-coefficient-method",
-    "1.716e-05",
-    "273.11",
-    "110.56",
-    "no",
-    "no",
-    "no",
-)
+solver.setup.materials.fluid["air"].density.option = "ideal-gas"
+
+solver.setup.materials.fluid["air"].viscosity.option = "sutherland"
+
+solver.setup.materials.fluid[
+    "air"
+].viscosity.sutherland.option = "three-coefficient-method"
+
+solver.setup.materials.fluid["air"].viscosity.sutherland.reference_viscosity = 1.716e-05
+
+solver.setup.materials.fluid["air"].viscosity.sutherland.reference_temperature = 273.11
+
+solver.setup.materials.fluid["air"].viscosity.sutherland.effective_temperature = 110.56
```

Comment:
We could create an alias for air e.g., 

```suggestion
air = solver.setup.materials.fluid["air"]

air.density.option = "ideal-gas"

air.viscosity.option = "sutherland"

viscosity_sutherland = air.viscosity.sutherland

viscosity_sutherland.option = "three-coefficient-method"

viscosity_sutherland.reference_viscosity = 1.716e-05

viscosity_sutherland.reference_temperature = 273.11

viscosity_sutherland.effective_temperature = 110.56
```
and so forth